Doc/library/webbrowser.rst

```diff
@@ -24,8 +24,17 @@ If the environment variable :envvar:`BROWSER` exists, it is interpreted as the
 :data:`os.pathsep`-separated list of browsers to try ahead of the platform
 defaults.  When the value of a list part contains the string ``%s``, then it is
 interpreted as a literal browser command line to be used with the argument URL
-substituted for ``%s``; if the part does not contain ``%s``, it is simply
-interpreted as the name of the browser to launch. [1]_
+substituted for ``%s``; if the value is a single word that refers to one of the
+already registered browsers this browser is added to the front of the search list; 
+if the part does not contain ``%s``, it is simply interpreted as the name of the 
```

Comment:
Lint fix: trim trailing spaces.

```suggestion
already registered browsers this browser is added to the front of the search list;
if the part does not contain ``%s``, it is simply interpreted as the name of the
```

Reply:
Those always get me, I need to investigate why my editor no longer strips trailing whitespace.